### PR TITLE
OCPBUGS-16815: EgressIP migration: use cloud egress-ipconfig instead of static values

### PR DESCRIPTION
**Pull request by kyrtapz**

### CODE REVIEW

**Comment by kyrtapz** [comment]
/pj-rehearse pull-ci-openshift-cluster-network-operator-master-e2e-aws-ovn-network-migration

**Comment by kyrtapz** [comment]
/pj-rehearse pull-ci-openshift-cluster-network-operator-master-e2e-aws-ovn-network-migration

File: ci-operator/step-registry/ovn/sdn-migration/feature-migration/validate-egress-ip/ovn-sdn-migration-feature-migration-validate-egress-ip-commands.sh [hunk 1]
```diff
@@ -23,25 +28,26 @@ validate_sdn_egressip_crs () {
...
-  echo "$EXPECT_HOSTSUBNET_EGRESS_CIDRS" | tee expected_egressCIDRs
-  echo "$EXPECT_HOSTSUBNET_EGRESS_IPS" | tee expected_hsn_egressIPs
-  echo "$EXPECT_NETNAMESPACE_EGRESS_IPS" | tee expected_nns_egressIPs
+  expected_egressCIDRs=$(oc get no  "$HOSTSUBNET_NAME" -o jsonpath="{.metadata.annotations.cloud\.network\.openshift\.io/egress-ipconfig}" | jq -r '.[].ifaddr.ipv4')
+  ip_part=$(echo "$expected_egressCIDRs" | cut -d'/' -f1)
+  expected_egressIPs="[\"${ip_part%.*}.5\"]"
+  expected_egressCIDRs="[\"$expected_egressCIDRs\"]"
 
-  if diff <(jq -S . current_egressCIDRs) <(jq -S . expected_egressCIDRs); then
+  if diff <(jq -S . current_egressCIDRs) <(jq -S <<< "$expected_egressCIDRs"); then
```
Comment (pliurh) [inline ci-operator/step-registry/ovn/sdn-migration/feature-migration/validate-egress-ip/ovn-sdn-migration-feature-migration-validate-egress-ip-commands.sh:36]: All rollback/reverse migration tests failed here. The variable in `jq -S <<< "$expected_egressCIDRs"` cannot be parsed correctly.

**Comment by kyrtapz** [comment]
/pj-rehearse pull-ci-openshift-cluster-network-operator-master-e2e-aws-sdn-network-reverse-migration

**Comment by kyrtapz** [comment]
/pj-rehearse pull-ci-openshift-cluster-network-operator-master-e2e-aws-sdn-network-reverse-migration

File: ci-operator/step-registry/ovn/sdn-migration/feature-migration/validate-egress-ip/ovn-sdn-migration-feature-migration-validate-egress-ip-commands.sh (hunk 1 above)
Comment (kyrtapz) [reply ci-operator/step-registry/ovn/sdn-migration/feature-migration/validate-egress-ip/ovn-sdn-migration-feature-migration-validate-egress-ip-commands.sh]: Fixed, jq is funny sometiems

**Comment by ricky-rav** [comment]
/lgtm

Thanks, Patryk!

**Pull request merged**